packages/tldraw/src/lib/shapes/shared/HyperlinkButton.tsx

```diff
@@ -19,6 +26,12 @@ export function HyperlinkButton({ url }: { url: string }) {
 			onPointerUp={stopEventPropagation}
 			title={url}
 			draggable={false}
+			onClick={(e) => {
+				if (selectToolActive && editor.inputs.shiftKey) {
+					e.preventDefault()
+					e.stopPropagation()
```

Comment:
We have our own stopEventPropagation and preventDefault helpers that let us track these event kills. (They're really hard to debug otherwise).